lib/jbuild

```diff
@@ -1,6 +1,7 @@
 (library
  ((name        tar)
   (public_name tar)
+  (wrapped false)
```

Comment:
do we really need this? API-wise, it's maybe better to use `Tar.Cstruct` than `Tar_cstruct`? This would just mean copying the doc strings from the new mli to `tar.mli`

Reply:
I started with that but didn't like that it was shadowing the `Cstruct` module. I'm happy to change it back, though, if it fits better with the project.

Reply:
that makes sense